frontend/packages/dev-console/yamls/pipelines/pipelinerun.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: tutorial-pipeline-run-1
+spec:
+  pipelineRef:
+    name: tutorial-pipeline
+  resources:
+    - name: source-repo
+      resourceRef:
+        name: skaffold-git
+    - name: web-image
+      resourceRef:
+        name: skaffold-image-leeroy-web
```

Comment:
pls add a newline

Reply:
You realize you're critiquing your own work, right? https://github.com/openshift/console/pull/3086/files#diff-2399a13e14dfba91856fded93033df41R14

Reply:
I understand, but only as part of the reviewing guidelines, I suggested the addition of the new lines. But I guess, it would not be easy with a cherry pick.